--- libfwbuilder-2.0.0/configure.in.orig	2004-08-01 17:35:21.269298000 +0200
+++ libfwbuilder-2.0.0/configure.in	2004-08-01 19:12:28.750386536 +0200
@@ -381,25 +381,10 @@
 
 
 
-dnl Check for bind specific headers and libraries
-dnl if they are present, use them, rather than ones
-dnl coming with libc.
-
-AC_CHECK_HEADER(bind/resolv.h, [
-  CPPFLAGS="$CPPFLAGS -I/usr/include/bind"
-  HAVE_RESOLV_H=yes
-] , [ AC_CHECK_HEADER(/usr/local/bind/include/resolv.h, [
-    CPPFLAGS="$CPPFLAGS -I/usr/local/bind/include/"
-    HAVE_RESOLV_H=yes
-  ], [ 
     AC_CHECK_HEADERS([resolv.h], [ HAVE_RESOLV_H=yes ], [], [
 #include <sys/types.h>
 #include <netinet/in.h> 
     ] )
-  ] )
-], [
-#include <netinet/in.h>
-] )
 
 
 
@@ -431,7 +416,6 @@
 AC_SUBST(LIB_RESOLV) 
 
 
-AC_CHECK_LIB(bind_r, res_init)
 
 
 
@@ -789,4 +773,4 @@
 
 echo "-------------------------------------------------------------------"
 
-. runqmake.sh
+. ./runqmake.sh
--- libfwbuilder-2.0.0/qmake.inc.in.orig	2004-07-03 08:36:52.000000000 +0200
+++ libfwbuilder-2.0.0/qmake.inc.in	2004-08-01 19:20:33.245732048 +0200
@@ -42,7 +42,7 @@
 	migration.path          =$(DDIR)@TEMPLATE_DIR@/migration
 	doc.path                =$(DDIR)@DOCDIR@
 
-	CONFIG                 += warn_on debug
+	CONFIG                 += warn_on
 	INCLUDEPATH            +=  @XML_CFLAGS_Q@ @XSLT_CFLAGS_Q@ @PTHREAD_CFLAGS_Q@ @OPENSSL_CFLAGS_Q@ @LWRES_CFLAGS_Q@
 	QMAKE_CFLAGS_DEBUG     += -Wno-unused-parameter
 	QMAKE_CFLAGS_RELEASE   += -Wno-unused-parameter
