--- libfwbuilder-2.0.2/configure.in	2004-08-29 02:38:44.000000000 +0000
+++ libfwbuilder-2.0.2.new/configure.in	2004-09-01 17:02:42.028616496 +0000
@@ -387,25 +387,10 @@
 
 
 
-dnl Check for bind specific headers and libraries
-dnl if they are present, use them, rather than ones
-dnl coming with libc.
-
-AC_CHECK_HEADER(bind/resolv.h, [
-  CPPFLAGS="$CPPFLAGS -I/usr/include/bind"
-  HAVE_RESOLV_H=yes
-] , [ AC_CHECK_HEADER(/usr/local/bind/include/resolv.h, [
-    CPPFLAGS="$CPPFLAGS -I/usr/local/bind/include/"
-    HAVE_RESOLV_H=yes
-  ], [ 
     AC_CHECK_HEADERS([resolv.h], [ HAVE_RESOLV_H=yes ], [], [
 #include <sys/types.h>
 #include <netinet/in.h> 
     ] )
-  ] )
-], [
-#include <netinet/in.h>
-] )
 
 
 
@@ -437,7 +422,6 @@
 AC_SUBST(LIB_RESOLV) 
 
 
-AC_CHECK_LIB(bind_r, res_init)
 
 
 
--- libfwbuilder-2.0.2/qmake.inc.in	2004-08-29 02:38:44.000000000 +0000
+++ libfwbuilder-2.0.2.new/qmake.inc.in	2004-09-01 17:03:10.374307296 +0000
@@ -43,7 +43,7 @@
 	migration.path          =$(DDIR)@TEMPLATE_DIR@/migration
 	doc.path                =$(DDIR)@DOCDIR@
 
-	CONFIG                 += warn_on debug
+	CONFIG                 += warn_on
 	INCLUDEPATH            += /usr/include @XML_CFLAGS_Q@ @XSLT_CFLAGS_Q@ @PTHREAD_CFLAGS_Q@ @OPENSSL_CFLAGS_Q@ @LWRES_CFLAGS_Q@
 	QMAKE_CFLAGS_DEBUG     += -Wno-unused-parameter
 	QMAKE_CFLAGS_RELEASE   += -Wno-unused-parameter
