--- libfwbuilder-0.10.7/src/fwbd/Makefile.in~	Fri Jul 19 20:58:22 2002
+++ libfwbuilder-0.10.7/src/fwbd/Makefile.in	Fri Jul 19 20:58:22 2002
@@ -43,7 +43,7 @@
 OPENSSL_CFLAGS = @OPENSSL_CFLAGS@
 OPENSSL_LIBS   = @OPENSSL_LIBS@
 
-CFLAGS = -Wall -g -I.. $(CPPFLAGS) $(GLIB_CFLAGS) $(OPENSSL_CFLAGS)
+CFLAGS = -Wall @CFLAGS@ -I.. $(CPPFLAGS) $(GLIB_CFLAGS) $(OPENSSL_CFLAGS)
 
 LDFLAGS = -g 
 LIBS = $(GLIB_LIBS) $(OPENSSL_LIBS) $(OTHER_LIBS)
--- libfwbuilder-0.10.7/src/fwbdclient/Makefile.in~	Fri Jul 19 20:58:58 2002
+++ libfwbuilder-0.10.7/src/fwbdclient/Makefile.in	Fri Jul 19 20:58:58 2002
@@ -23,7 +23,7 @@
 
 GLIB_CFLAGS     = @GLIB_CFLAGS@
 
-CFLAGS  = -Wall  -g -I.. -I$(top_srcdir) $(XML_CFLAGS) $(GLIB_CFLAGS) $(OPENSSL_CFLAGS)
+CFLAGS  = -Wall @CFLAGS@ -I.. -I$(top_srcdir) $(XML_CFLAGS) $(GLIB_CFLAGS) $(OPENSSL_CFLAGS)
 LDFLAGS = -L$(libdir) -L../fwbd/.libs -lfwbd $(LIBS)
 
 SRC=fwbdclient.c
--- libfwbuilder-0.10.7/src/fwbuilder/Makefile.in~	Fri Jul 19 20:59:45 2002
+++ libfwbuilder-0.10.7/src/fwbuilder/Makefile.in	Fri Jul 19 20:59:45 2002
@@ -54,7 +54,7 @@
 
 OTHER_LIBS = @LIBS@
 
-CFLAGS = -Wall -g -I.. $(CPPFLAGS) $(GLIB_CFLAGS) \
+CFLAGS = -Wall @CFLAGS@ -I.. $(CPPFLAGS) $(GLIB_CFLAGS) \
 	$(XML_CFLAGS) $(XSLT_CFLAGS) $(OPENSSL_CFLAGS) $(SIGC_CFLAGS)
 
 LDFLAGS = -g 
--- libfwbuilder-0.10.7/src/fwcompiler/Makefile.in~	Fri Jul 19 21:00:21 2002
+++ libfwbuilder-0.10.7/src/fwcompiler/Makefile.in	Fri Jul 19 21:00:21 2002
@@ -53,7 +53,7 @@
 
 OTHER_LIBS =  @LIBS@
 
-CFLAGS =  -g -I.. -I$(top_srcdir) $(CPPFLAGS) $(GLIB_CFLAGS) \
+CFLAGS =  @CFLAGS@ -I.. -I$(top_srcdir) $(CPPFLAGS) $(GLIB_CFLAGS) \
         $(LIBFWBUILDER_CFLAGS) $(XML_CFLAGS) $(XSLT_CFLAGS)
 
 
