diff -urN aaa/libfwbuilder-1.0.1/configure.in libfwbuilder-1.0.1/configure.in
--- aaa/libfwbuilder-1.0.1/configure.in	2003-08-03 20:27:52.000000000 +0200
+++ libfwbuilder-1.0.1/configure.in	2003-09-09 12:16:08.000000000 +0200
@@ -20,13 +20,6 @@
 
 EXTENDED_PATH="$PATH:/usr/local/bin:/opt/bin:/opt/gnome/bin:/sw/bin"
 
-dnl
-dnl we do not want to build .a libraries. We could supply "--disable-static" option
-dnl to the call for configure, but it requires changes in the package configuration
-dnl files for all OS and that is just too much to remember.
-dnl
-enable_static=no
-
 LIBFWBUILDER_FORMAT_VERSION="${LIBFWBUILDER_VERSION}"
 
 AC_DEFINE_UNQUOTED(LIBFWBUILDER_FORMAT_VERSION, "${LIBFWBUILDER_FORMAT_VERSION}")
