diff -uN libfwbuilder-2.1.5.orig/configure.in libfwbuilder-2.1.5/configure.in
--- libfwbuilder-2.1.5.orig/configure.in	2006-08-20 18:36:03.000000000 +0200
+++ libfwbuilder-2.1.5/configure.in	2006-08-30 16:59:17.000000000 +0200
@@ -391,25 +391,10 @@
 
 
 
-dnl Check for bind specific headers and libraries
-dnl if they are present, use them, rather than ones
-dnl coming with libc.
-
-AC_CHECK_HEADER(bind/resolv.h, [
-  CPPFLAGS="$CPPFLAGS -I/usr/include/bind"
-  HAVE_RESOLV_H=yes
-] , [ AC_CHECK_HEADER(/usr/local/bind/include/resolv.h, [
-    CPPFLAGS="$CPPFLAGS -I/usr/local/bind/include/"
-    HAVE_RESOLV_H=yes
-  ], [ 
-    AC_CHECK_HEADERS([resolv.h], [ HAVE_RESOLV_H=yes ], [], [
+   AC_CHECK_HEADERS([resolv.h], [ HAVE_RESOLV_H=yes ], [], [
 #include <sys/types.h>
 #include <netinet/in.h> 
     ] )
-  ] )
-], [
-#include <netinet/in.h>
-] )
 
 
 dnl
--- libfwbuilder-3.0.5-orig/qmake.inc.in	2009-07-15 14:02:41.514653425 +0200
+++ libfwbuilder-3.0.5/qmake.inc.in	2009-07-15 14:03:22.750469971 +0200
@@ -29,7 +29,7 @@ unix {
          migration.path = @TEMPLATE_DIR@/migration
          doc.path = @DOCDIR@
 
-         CONFIG += warn_on debug
+         CONFIG += warn_on
          QMAKE_CFLAGS_DEBUG += -Wno-unused-parameter
          QMAKE_CFLAGS_RELEASE += -Wno-unused-parameter
          QMAKE_CXXFLAGS_DEBUG += -Wno-unused-parameter @CXXFLAGS@
